File: backend/src/main/java/by/fpmibsu/findafriend/controller/commands/shelters/UpdateShelterCommand.java
package by.fpmibsu.findafriend.controller.commands.shelters;

import by.fpmibsu.findafriend.application.mediatr.Request;
import by.fpmibsu.findafriend.entity.Place;

public class UpdateShelterCommand extends Request<Boolean> {
    public int shelterId;
    public String name;
    public int placeId;
    public String address;
    public String website;
    public UpdateShelterCommand() {}

    public UpdateShelterCommand(int shelterId, String name, int placeId, String address, String website) {
        this.shelterId = shelterId;
        this.name = name;
        this.placeId = placeId;
        this.address = address;
        this.website = website;
    }
}

File: BackendNET/Backend/Commands/Shelters/UpdateShelter/UpdateShelterCommand.cs
using MediatR;

namespace Backend.Commands.Shelters.UpdateShelter;

public class UpdateShelterCommand : IRequest<bool>
{
    public int ShelterId { get; set; }
    public string? Name { get; set; }
    public string? Address { get; set; }
    public string? Website { get; set; }
    public int? PlaceId { get; set; }
}